packages/core/src/layout/SignInPage/index.ts

```diff
@@ -14,4 +14,5 @@
  * limitations under the License.
  */
 
+export type { SignInConfig } from './types';
```

Comment:
Thinking we want to rename this if we're gonna export it, something like `SignInPageProvider`? Or `SignInPageProviderConfig` / `SignInProviderConfig`, seeing as this is not the configuration for "sign-in".